pkg/network/master/egressip.go

```diff
@@ -197,9 +197,7 @@ func (eim *egressIPManager) maybeDoUpdateEgressCIDRs() (bool, error) {
 }
 
 const (
-	pollInterval   = 5 * time.Second
```

Comment:
@kyrtapz qq on this particular commit: the reason for moving it is to remove the duplication that we observe in this file and in pkg/network/node/egressip.go, correct?